lofty.com.website.json

```diff
@@ -82,18 +75,20 @@
       "txtConflictMatchingMode": "All"
     },
     {
-      "type": "SPFM",
+      "type": "TXT",
       "groupId": "email",
-      "host": "@",
+      "host": "mail",
       "ttl": 3600,
-      "spfRules": "include:amazonses.com"
+      "data": "v=spf1 %spfRules% ~all",
+      "essential": "OnApply"
     },
     {
-      "type": "SPFM",
+      "type": "TXT",
       "groupId": "email",
-      "host": "mail",
+      "host": "@",
       "ttl": 3600,
-      "spfRules": "include:amazonses.com"
+      "data": "v=spf1 %spfRules% ~all",
```

Comment:
Replacing SPFM on APEX is actually a bad idea. Do it only if you plan to make a full merge of SPF on your side and even if you do that there are field reports of very bad side effects.
Normally you'd be better off sticking to SPFM. This approach is just a workaround (and actually in the spec how DNS provider may emulate SPFM).

Reply:
ok, I will use SPFM record on APEX.